sys/crypto/dragon.c

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (C) 2015 Jan Wagner <mail@jwagner.eu>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
```

Comment:
Dragon algorithm is made freely available to all users, at their own risk, same for other ciphers

http://www.ecrypt.eu.org/stream/phase3ip.html#dragon